web/docs/introduction/migrate-from-11.md

```diff
@@ -158,25 +157,40 @@ You can easily migrate your old Wasp project to the new structure by following a
 series of steps. Assuming you have a project called `foo` inside the
 directory `foo`, you should:
 
-0. Make sure to backup or save your project before starting the procedure (e.g.,
+0. Install the latest `0.12.x` version of Wasp.
+  ```bash
+  curl -sSL https://get.wasp-lang.dev/installer.sh | sh -s
+  ```
+1. Make sure to backup or save your project before starting the procedure (e.g.,
    by committing it to source control or creating a copy).
-1. Install the latest `0.12.x` version of Wasp.
-2. Rename your project's root directory to something like `foo_old`.
-3. Create a new project by running `wasp new foo`.
-4. Delete all files of `foo/src` except `vite-env.d.ts`.
-5. If `foo_old/src/client/public` exists and contains any files, copy those files into
+2. Position yourself in the terminal in the directory that is a parent of your wasp project directory (so one level above: if you do `ls`, you should see your wasp project dir listed).
+3. Run the migration script (replace `foo` at the end with the name of your Wasp project directory) and follow the instructions:
+  ```
+    curl -sSL https://raw.githubusercontent.com/wasp-lang/wasp/main/waspc/tools/migrate-0-11-to-0-12.js | node - foo
+  ```
+
+<details>
```

Comment:
I moved the manual steps into this thing that can be expanded. They are the same otherwise, except for one small update in the one of the latest steps (I specified some files a bit better after I learned from Filip what exactly is required there).